src/components/pages/supporters/Supporter.vue

```diff
@@ -1,69 +1,69 @@
 <template>
-	<div class="accordion-item" v-bind:class="[ isOpen ? 'accordion' : '' ]">
-		<div @click="toggle()">
-			<div v-bind:class="[ isOpen ? 'has-text-primary has-text-weight-bold' : 'accordion-heading', 'icon-inline', 'accordion-title' ] ">
-				<span class="container columns is-mobile">
-					<span class="column">{{ content.name }}</span>
-					<span class="column is-narrow">
-						<span class="is-nowrap-whitespace is-narrow has-padding-right-18">{{ content.amount }}</span>
-						<span class="icon-aligned">
-							<ArrowUp v-if="isExpandable && isOpen"/>
-							<ArrowDown v-else-if="isExpandable && !isOpen"/>
-						</span>
-					</span>
-				</span>
+	<div class="accordion-item" :class="[isOpen ? 'accordion' : '']">
+		<div @click="toggle">
+			<div :class="[isOpen ? 'has-text-primary has-text-weight-bold' : 'accordion-heading', 'icon-inline', 'accordion-title']">
+        <span class="container columns is-mobile">
+          <span class="column">{{ content.name }}</span>
+          <span class="column is-narrow">
+            <span class="is-nowrap-whitespace is-narrow has-padding-right-18">{{ content.amount }}</span>
+            <span class="icon-aligned">
+              <ArrowUp v-if="isExpandable && isOpen"/>
+              <ArrowDown v-else-if="isExpandable && !isOpen"/>
+            </span>
+          </span>
+        </span>
 			</div>
 		</div>
 		<div v-show="isOpen" v-html="content.comment" class="accordion-content"></div>
 	</div>
 </template>
 
+<script setup lang="ts">
+import { Supporter } from '@src/view_models/supporters';
+import { computed, ref } from 'vue';
+import { emit } from 'process';
```

Comment:
`emit` is not imported, you call `const emit = defineEmits(['supporter-openened', 'supporter-closed'])` instead. `defineEmits` are so-called "vue compiler macros", functions that are "just there" and don't need to be imported. See https://vuejs.org/api/sfc-script-setup.html#defineprops-defineemits